source/docs/articles/sites/migrate/wordpress-site-networks.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Migrate to Pantheon: WordPress Site Networks
+description: Learn how to import a WordPress Site Network into Pantheon.
+keywords: import, importing site, pantheon, new site, large site, distro, upstream, git history
+---
+
+## Requirements
+
+* [Download](http://git-scm.com/downloads) and install [Git](/docs/articles/local/starting-with-git/)
+* [Rsync or SFTP Client](https://pantheon.io/docs/articles/local/rsync-and-sftp/)
+* [MySQL Client](https://pantheon.io/docs/articles/local/accessing-mysql-databases/)
+* A Pantheon Employee must create a [WordPress Site Network](/docs/articles/wordpress/site-networks/) for you.
+* You have [created a compatible site archive](/docs/articles/sites/migrate/export-an-existing-wordpress-site#manually-create-separate-site-archives)
+
+## Import the Codebase
+
+**Codebase** - all executable code, including core, plugins, themes, and libraries. Stored in the `~/code` directory.
+
+When you [created a compatible site archive](/docs/articles/sites/migrate/export-an-existing-wordpress-site#manually-create-separate-site-archives), you may have needed to move blog-specific uploads directories located outside of `wp-content/uploads` into `wp-content/uploads`, and replace the original directories with symlinks to their new homes. If you haven't done so, please do so now.

```

Comment:
In more recent versions of WordPress multisite, blog-specific uploads are stored in `wp-content/uploads/sites/<id>`. Might be worth mentioning to the user what they could be looking for.